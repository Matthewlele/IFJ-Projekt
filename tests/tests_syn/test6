df foo ()
  x = 50
  y = 40
  z = (x/y)*y+x
end

retVal = foo
print retVal
