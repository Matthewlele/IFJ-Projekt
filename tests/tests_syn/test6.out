expected SEMANTIC ERR: row 1
